src/main/kotlin/com/mjucow/eatda/common/config/SwaggerConfig.kt

```diff
@@ -0,0 +1,20 @@
+package com.mjucow.eatda.common.config
+
+import io.swagger.v3.oas.models.Components
+import io.swagger.v3.oas.models.OpenAPI
+import io.swagger.v3.oas.models.info.Info
+import org.springframework.context.annotation.Bean
+import org.springframework.context.annotation.Configuration
+
+@Configuration
+class SwaggerConfig {
+    @Bean
+    fun swaggerApi(): OpenAPI = OpenAPI()
```

Comment:
타입 변환이 따로 없다면 명시적으로 타입을 선언해주지 않아도 될 것 같아요!
```suggestion
    fun swaggerApi() = OpenAPI()
```